pkg/cmd/roachtest/tests/import.go

```diff
@@ -355,7 +355,7 @@ func registerImportDecommissioned(r registry.Registry) {
 		// Decommission a node.
 		nodeToDecommission := 2
 		t.Status(fmt.Sprintf("decommissioning node %d", nodeToDecommission))
-		c.Run(ctx, option.WithNodes(c.Node(nodeToDecommission)), `./cockroach node decommission --self --wait=all --url={pgurl:2}`)
+		c.Run(ctx, option.WithNodes(c.Node(nodeToDecommission)), fmt.Sprintf(`./cockroach node decommission --self --wait=all --port={pgport:%d} --certs-dir=%s`, nodeToDecommission, install.CockroachNodeCertsDir))
```

Comment:
Why was this change needed?

Reply:
`cockroach node` doesn't support non root authentication since it's an RPC connection. All the other `cockroach node` calls already used this `--port` and `--certs-dir` pattern, this was just the outlier.